tui_main.c

```diff
@@ -2335,21 +2335,21 @@ int main(int argc, char** argv) {
     switch (key) {
     case KEY_UP:
     case CTRL_PLUS('k'):
-      ged_dir_input(&ged_state, Ged_dir_up);
+	ged_dir_input(&ged_state, Ged_dir_up, 1);
       break;
     case CTRL_PLUS('j'):
-    case KEY_DOWN:
-      ged_dir_input(&ged_state, Ged_dir_down);
+    case KEY_DOWN://HERHER
```

Comment:
what does this `//HERHER` means ?

Reply:
Argh, sorry, sorry, sorry. old helper code, should I remove it and do another PR?